templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig

```diff
@@ -38,26 +39,42 @@
 
                 {% if feedback.success_message is defined %}
                     <p class="flash flash-confirm">
-                        <i class="fa fa-check-circle" aria-hidden="true"></i>
+                        <i
+                            class="fa fa-check-circle"
+                            aria-hidden="true"></i>
                         {{ feedback.success_message }}
                     </p>
                 {% endif %}
 
                 {% if feedback.fail_message is defined %}
                     <p class="flash flash-error">
                         {% for message in feedback.fail_message %}
-                            <i class="fa fa-exclamation-circle" aria-hidden="true"></i> {{ message }}<br>
+                            <i
+                                class="fa fa-exclamation-circle"
+                                aria-hidden="true"></i>
+                            {{ message }}
+                            <br>
                         {% endfor %}
                     </p>
                 {% endif %}
 
-                <form id="newUser" method="post" action="{{ path("DemosPlan_user_complete_data") }}" data-dp-validate novalidate>
+                <form
+                    id="newUser"
+                    method="post"
+                    action="{{ path("DemosPlan_user_complete_data") }}"
+                    data-dp-validate novalidate>
 
-                    <h3 class="u-mt-2 u-mb-0_75">Nutzerbezogene Daten</h3>
+                    <h3 class="u-mt-2 u-mb-0_75">
+                        {{ "customer.data"|trans }}
+                    </h3>
 
                     <label class="u-mb-0_5">
                         {{ "email"|trans }}
-                        <input type="text" class="layout__item" value="{% if user is defined %}{{- user.email|default("-") -}}{% endif %}" disabled>
+                        <input
+                            type="text"
+                            class="layout__item"
+                            value="{% if user is defined %}{{- user.email|default("-") -}}{% endif %}"
+                            disabled>
```

Comment:
why is this input field always disabled, I wonder if we need this if it is always disabled.